frontend/src/components/ExchangePage.vue

```diff
@@ -206,25 +207,20 @@ export default {
   mixins: [getSecureExchangeContactMixin],
   data() {
     return {
+      initialLoad: true,
       selectedItem: 0,
-      pageNumber: 1,
-      pageSize: 25,
       totalRequests: 0,
-      itemsPerPageOptions: [10, 15, 25, 50, 100],
-      loadingTable: false,
-      dateMenu: false,
-      headerSearchParams: {
+      searchParams: {
         sequenceNumber: '',
         contact: '',
         subject: '',
         createDate: [],
-        status: '',
+        secureExchangeStatusCode: '',
         reviewer: ''
       },
-      headerSortParams: {
-        currentSort: 'createDate',
-        currentSortDir: true
-      },
+      itemsPerPageOptions: [10, 15, 25, 50, 100],
```

Comment:
I think we should discuss as a team on the number of items per page.

Reply:
Yeah we can. The ticket just says 25 items. I just left the additional options in there.